.travis/opensuse.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -xe
-${DOCKER_EXEC} zypper -n ar -f -G http://download.opensuse.org/repositories/devel:/tools/openSUSE_13.2/devel:tools.repo
+OS_VERSION=$(${DOCKER_EXEC} cat /etc/os-release | awk -F[=\"] '/PRETTY_NAME=/ {print $3}')
```

Comment:
@arungrover I don't think 13.2 will come back again EVER as it is reached end of life. and I checked PRETTY_NAME on 42.1, 42.2 (latest) and tumbleweed and it works there. So I don't think we need any change here.